optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -40,7 +38,7 @@ public void initialization() {
 
         StringBuilder sb = new StringBuilder(100);
         fact.printInitialization(sb);
-        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();\n", sb.toString());
+        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();" + System.lineSeparator(), sb.toString());
```

Comment:
@yurloc Any idea why it's not only a "\n" on Windows too? Ideally all .java files only have linux line endings, regardless on what platform they are generated (we already enforce this for our normal .java files in .gitattributes)